translations/data/home.json

```diff
@@ -1,125 +1,143 @@
 {
     "intro_what_is_fediverse": {
         "en-us": "What is <mark>fediverse</mark>?",
+	"de-de": "Was ist das <mark>Fediverse</mark>?",
         "sk": "Čo je <mark>fediverse</mark>?",
         "fr": "Qu'est-ce que le <mark>fediverse</mark> ?",
         "ru": "Что такое <mark>fediverse</mark>?",
         "ar": "ما هو <mark>الفديفيرس</ mark>؟"
     },
     "intro_btn_explore": {
         "en-us": "Explore",
+	"de-de": "Erkunden",
         "sk": "Povedz mi",
         "fr": "Explorer",
         "ru": "Обзор",
         "ar": "استكشف"
     },
     "intro_analogy": {
         "en-us": "A good analogy for the fediverse is <mark>email</mark>.",
+	"de-de": "Als Analogie zum Fediverse bietet sich <mark>E-Mail</mark> an.",
         "sk": "Fediverse sa najviac podobá <mark>emailu</mark>.",
         "fr": "Une analogie utile pour le fediverse est l'<mark>email</mark>.",
         "ru": "Хорошей аналогией для fediverse можно назвать <mark>электронную почту</mark>.",
         "ar": "<mark> البريد الإلكتروني </ mark> هو تشبيه جيد بالنسبة إلى الفديفيرس."
     },
     "intro_email": {
         "en-us": "You're probably using Gmail. Or perhaps Outlook. Or Yahoo Mail.",
+	"de-de": "Du benutzt vielleicht gmail. Oder GMX. Oder Web.de ...",
         "sk": "Pravdepodobne používaš Gmail. Alebo Zoznam.sk. Možno Centrum.sk.",
         "fr": "Vous utilisez probablement Gmail. Ou peut-être Outlook. Ou Yahoo Mail.",
         "ru": "Вы, скорее всего, используете Gmail. Или, может, Outlook. Или Yahoo Mail.",
         "ar": "من المحتمل أنك تستخدم Gmail أو ربما Outlook أو بريد Yahoo."
     },
     "intro_anyone": {
         "en-us": "Regardless of your choice, you can send an email to <mark>anyone</mark> in the world without knowing or caring what service they use.",
+	"de-de": "Unabhängig davon, für welchen E-Mail-Anbieter Du Dich entschieden hast, kannst Du an <mark>jeden</mark> auf der Welt E-Mails schicken, ohne Dich darum kümmern zu müssen, welchen Anbieter der Empfänger nutzt.",
         "sk": "Bez ohľadu na tvoju voľbu, môžeš poslať email <mark>hocikomu</mark> na celom svete.",
         "fr": "Indépendamment du choix que vous avez effectué, vous pouvez adresser un email à toute personne dans le monde, sans avoir à vous soucier du service qu'elle utilise.",
         "ru": "Независимо от вашего выбора, вы всегда можете отправить письмо <mark>любому</mark> человеку в мире, не зная и не думая о том, каким сервисом они пользуются.",
         "ar": "بغض النظر عن اختيارك ، يمكنك إرسال بريد إلكتروني إلى <mark> أي شخص </ mark> في العالم دون معرفة أو الاهتمام بالخدمة التي يستخدمونها." 
     },
     "intro_btn_neat": {
         "en-us": "Neat",
+	"de-de": "Prima - weiter",

```

Comment:
` - weiter` is not part of the English (I guess it is the major/main language?).

Reply:
Yes, I think Prima should be enough here.